### PR TITLE
RK-14337 - add new rookout chart

### PR DESCRIPTION
**Pull request by alexeygutkin**

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -25,14 +25,14 @@ jobs:
       - name: Install helm 🚀
         uses: azure/setup-helm@v1
         with:
-          version: '2.16.7'
+          version: '3.10.0'
 
       - name: Validate Charts ✅️
         run: |
           helm version -c
-          helm init -c --stable-repo-url https://charts.helm.sh/stable
```
Comment (idanrookout) [inline .github/workflows/publish.yml:33]: we don't need it anymore?

Comment (alexeygutkin) [reply .github/workflows/publish.yml:33]: init deprecated in helm 3

File: README.md [hunk 2]
```diff
@@ -18,6 +18,8 @@
 
 [datastore](https://github.com/rookout/helm-charts/tree/master/charts/datastore) - Controller required
 
+[rookout](https://github.com/rookout/helm-charts/tree/master/charts/rookout) - Rookout's hybrid deployment components (ETL Controller + Datastore)
```
Comment (lirook) [inline README.md:21]: Add a (deprecated) on line 19 and 17

**Review by lirook — CHANGES_REQUESTED** [review]

**Comment by lirook** [comment]
I left just one note, other than that if this is a copy paste from the other branch, it looks good to me

**Review by idanrookout — DISMISSED** [review]

**Review by idanrookout — APPROVED** [review]

**Review by gosharo — APPROVED** [review]

**Review by idanrookout — APPROVED** [review]

**Review by lirook — APPROVED** [review]

**Pull request merged**